package.json

```diff
@@ -35,23 +35,26 @@
     "bootstrap-sass": "^3.3.6",
     "classnames": "^2.2.3",
     "crypto-js": "^3.1.5",
+    "cyverse-ui": "^0.*.*",
```

Comment:
Do we really want to accept any `cyverse-ui` version like this? Is the module in that much flux?